pkg/repository/prisma/tenant_invite.go

```diff
@@ -26,6 +27,22 @@ func (r *tenantInviteRepository) CreateTenantInvite(tenantId string, opts *repos
 		return nil, err
 	}
 
+	til, err := r.client.TenantInviteLink.FindFirst(
```

Comment:
Let's also set a max limit of 100 outstanding invites, which can be configured with an environment variable?